app/build.gradle

```diff
@@ -188,60 +188,60 @@ dependencies {
     implementation project(':third_party:subsampling-scale-image-view')
     implementation project(':third_party:glide:annotation')
     implementation project(':third_party:glide:library')
-    annotationProcessor Libs.glide_compiler
-    kapt Libs.glide_compiler
+    annotationProcessor "com.github.bumptech.glide:compiler:${Versions.glide}"
```

Comment:
Should we merge the #1808 first then rebase it?

Reply:
I think I'll do the reverse, I'll rebase 1808 since rebasing that one is simpler :P